lib/registrant.js

```diff
@@ -158,9 +158,11 @@ Registrant.prototype.updateThingData = function(identity, data, schemaIndex, opt
  * @returns {Promise} that resolves with transaction hash of the operation.
  * @note Sends transaction, costs Ether.
  */
-Registrant.prototype.deleteThing = function(identity) {
+Registrant.prototype.deleteThing = function(identity, opts = {}) {
+  const defaults = { gas: 4000000};
```

Comment:
ok, so what would you set the gas limit to then? And I suppose we should change the defaults for `createThings`, and `updateThing` as well? Although keep in mind these are all defaults and can change if specified in the function invocation